cmd/prow-job-dispatcher/main.go

```diff
@@ -656,29 +685,31 @@ func runAsDaemon(o options, promVolumes *prometheusVolumes) {
 	}
 	defer watcher.Close()
 
-	go func() {
+	go func(config string) {
```

Comment:
Unfortunately, watchers do not work good with git-sync, they completely do not catch events as git sync is removing and re-adding everything. Simple loop with 30 secs delay should handle this.

Reply:
This comment is probably worthy of being immortalized in the code itself :)